packages/client/src/layouts/authenticated-layout.vue

```diff
@@ -504,7 +508,12 @@ function openSettings() {
         notifyError(t("common.genericErrorMessage"));
       }
     } else {
-      // TODO: perform annual reset
+      await resetRetailLocation({
+        input: {
+          retailLocationId: selectedLocation.value.id,
+        },
+      });
+      // window.location.reload();
```

Comment:
I am not sure if we should reload the page directly or after displaying a notification for a few seconds. We might also want to do this differently. But, we definitely need to do something like this to reset the caches. Please let me know what you think.

Reply:
I think that resetting the cache is the best thing we can do.
Trying to deal with updating it all seems like unnecessary complex to me for something that happens just once per year.
Thou have my blessing to proceed like this

Reply:
Do you mean reset all the caches but do not refresh the page or display a notification?

Reply:
Sorry, I've seen your reply only now.
I meant that I am more than fine with refreshing the page entirely, since this operation will happen once a year.

Reply:
Done 👍